test/integration/shoots/sshrotation/sshkey_rotation.go

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+	Overview
+		- Tests ssh-keypair rotation
+
+	Test: Annotate Shoot with "gardener.cloud/operation" = "rotate-ssh-keypair"
+	Expected Output
+		- Current ssh-keypair should be rotated
+		- Current ssh-keypair should become "ssh-keypair.old" post rotation
+ **/
+
+package sshrotation
+
+import (
+	"context"
+	"time"
+
+	corev1beta1 "github.com/gardener/gardener/pkg/apis/core/v1beta1"
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/test/framework"
+
+	"github.com/onsi/ginkgo"
+	"github.com/onsi/gomega"
```

Comment:
Can we add a `.`, so that we can refer to the imported functions without a qualifier?